postgres-appliance/configure_spilo.py

```diff
@@ -42,7 +66,9 @@ def write_certificates(environment):
         output, _ = p.communicate()
         logging.debug(output)
 
-    os.chmod(environment['SSL_PRIVATE_KEY_FILE'], 0o600)
+    gid = os.stat(environment['PGHOME']).st_gid
+    os.chmod(environment['SSL_PRIVATE_KEY_FILE'], 0o640)
+    os.chown(environment['SSL_PRIVATE_KEY_FILE'], -1, gid)
```

Comment:
I thought it would prevent the postgres os-user from changing the file; but that assumption is wrong, as the directory is owned by postgres.
Switching to uid makes more sense.